ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -54,11 +54,11 @@ def _compute_red_zone(self):
             if rec.replenish_method in ['replenish', 'min_max']:
```

Comment:
You have to split this, for `replenish` the code is good. For `min_max` as we discussed yesterday they changed the computation of red zone on the 2nd edition. Now it is RZ(min_max) = adu * dlt * (1 + variability factor)

Reply:
> You have to split this, for replenish the code is good

Here I meant the current code without the changes on this PR